### PR TITLE
homework_8

### PR DESCRIPTION
**Pull request by EvgeniyPon**
Hi

### CODE REVIEW

File: Lesson_8_Dictionary/Homework_8.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          'manager' : ["Anastasia", "Dmitry", 'asd'], 'developer': ["Evgeniy"]}
+name = input('What is yor name?')
+name_roles = []
+for key, value in roles.items():
+    if name in value:
+        name_roles.append(key)
+if name_roles:
+    print(f'Hello, {' & ' .join(name_roles)}!')
+else:
+    print('Hello, Guest!')
```
Comment (MarinaZelenska) [inline Lesson_8_Dictionary/Homework_8.py:12]: Working) but can be better
image

File: Lesson_8_Dictionary/Homework_8.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+# Task_2 add dictionary to string
+params = {'v': 'some_id', 'list': 'some_list', 'index': '6', 't': '0s'}
+initial_str = 'https://www.youtube.com/watch?'
+result_link = initial_str
+for key, values in params.items():
+    result_link += f'{key}={values}&'
+result_link = result_link[:-1:]
```
Comment (MarinaZelenska) [inline Lesson_8_Dictionary/Homework_8.py]: result_link = result_link.strip('&')

File: Lesson_8_Dictionary/Homework_8.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+print(result_link)
```
Comment (MarinaZelenska) [inline Lesson_8_Dictionary/Homework_8.py:33]: You should use formatting string all time

File: Lesson_8_Dictionary/Homework_8.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        result_dict[_key] += 1
+    else:
+        result_dict[_key] = 1
+print('quantity of repeated symbols:')
+row = 0
+for key, value in result_dict.items():
+    print(f'{key}: {value}', end='\t')
+    row += 1
+    if row % 5 == 0:
+        print()
```
Comment (MarinaZelenska) [inline Lesson_8_Dictionary/Homework_8.py:49]: Working) but can be better
image

**Review by MarinaZelenska — APPROVED** [review]

File: Lesson_8_Dictionary/Homework_8.py (hunk 1 above)
Comment (EvgeniyPon) [reply Lesson_8_Dictionary/Homework_8.py:12]: thx for checking & comment! 
I have been started from this way,  but that code is not consider about more than one roles and case of non title name

File: Lesson_8_Dictionary/Homework_8.py (hunk 2 above)
Comment (EvgeniyPon) [reply Lesson_8_Dictionary/Homework_8.py]: got it

File: Lesson_8_Dictionary/Homework_8.py (hunk 3 above)
Comment (EvgeniyPon) [reply Lesson_8_Dictionary/Homework_8.py:33]: will correct

**Pull request merged**